attachments/forms.py

```diff
@@ -80,3 +75,20 @@ def formfield(self, prop, field_class=None, **kwargs):
         defaults.update(kwargs)
         field = field_class(**defaults)
         return field
+    
+    @staticmethod
+    def get_form_data_from_session_data(session_data):
+        """
+        This handles converting session.data to a data object that the property form can use for initialization.
+        Deserialized session data will be lists containing only one item, which is the value we want in the field.
+        Note: This will need to be updated if widgets that can support multiple selections are added.
+        """
+        form_data = {}
+        if session_data is not None:
+            for key, val in session_data.iteritems():
```

Comment:
There's no iteritems in Python 3, probably should just use items and accept it will be a little less memory-efficient in Python 2.